protocol/app/flags/flags.go

```diff
@@ -74,5 +97,13 @@ func GetFlagValuesFromOptions(
 		result.DdTraceAgentPort = v
 	}
 
+	if v, ok := appOpts.Get(GrpcAddress).(string); ok {
+		result.GrpcAddress = v
+	}
+
+	if v, ok := appOpts.Get(GrpcEnable).(bool); ok {
```

Comment:
Will this logic be triggered if `grpc.enable` is set to `false` in the `config.toml`?